Formula/stanford-parser.rb

```diff
@@ -1,10 +1,8 @@
 class StanfordParser < Formula
   desc "Statistical NLP parser"
   homepage "https://nlp.stanford.edu/software/lex-parser.shtml"
-  url "https://nlp.stanford.edu/software/stanford-parser-full-2018-10-17.zip"
-  version "3.9.2"
-  sha256 "92d852af54c0727c2367b9ce267c53bf08f1551a08ec5dd92c357b8cc7b2bcd9"
-  revision 1
+  url "https://nlp.stanford.edu/software/stanford-parser-4.2.0.zip"
+  sha256 "8c2110c78f7f82b66bcf91089a18e415669eda4346bbd9a6e3bc2bde63e5fed1"
```

Comment:
```suggestion
  sha256 "8c2110c78f7f82b66bcf91089a18e415669eda4346bbd9a6e3bc2bde63e5fed1"
  license "GPL-2.0-or-later"
```
From `README.txt`:
```
// This program is free software; you can redistribute it and/or
// modify it under the terms of the GNU General Public License
// as published by the Free Software Foundation; either version 2
// of the License, or (at your option) any later version.
```
If you make this change, squash your commits together, please. Thanks!

Reply:
Sounds good!